.changeset/fluffy-lions-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/oauth-providers': major
```

Comment:
This should be a `patch`.

Reply:
@nakasyou any updates on this PR ? If not available I would like to get back to it 👍 .

Reply:
Sorry for very late response, I fixed it.